drupal/templates/deployment.yaml

```diff
@@ -19,8 +19,8 @@ spec:
     spec:
       containers:
       - name: {{ template "fullname" . }}
-        image: "bitnami/drupal:{{ .Values.imageTag }}"
-        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+        image: "{{ .Values.image.repo }}:{{ .Values.image.tag }}"
```

Comment:
What do you think if instead of splitting it make values.image to include the tag as well if wanted?

That seems to me easier to understand by user since it is the same way people configure docker-compose files, etc.